salt/modules/idem.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+#
+# Author: Tyler Johnson <tjohnson@saltstack.com>
+#
+
+"""
+Idem Support
+============
+
+This module provides access to idem execution modules
```

Comment:
The docstring for a new module [should have](https://docs.saltstack.com/en/master/topics/development/conventions/style.html#docstring-conventions) the `.. versionadded:: Magnesium` tag